api/api/V1/Processing.ts

```diff
@@ -521,6 +691,93 @@ export default function (app: Application, baseUrl: string) {
     },
   );
 
+  /**
+   * @api {post} /api/v1/processing/:id/tracks/:trackId/tagsBulk Add tags to track
```

Comment:
What's the use case for tagsBulk as well as tracksAndTags?  Is this mostly for re-processing?